src/components/inputs/z-input/styles-general.css

```diff
@@ -67,7 +67,7 @@ textarea:focus:focus-visible {
 /* HOVER */
 input:hover,
 .textareaWrapper:hover {
-  border-width: var(--border-size-medium);
+  outline: var(--border-size-small) solid var(--color-surface04);
```

Comment:
Mi sembra che l'outline non dia l'effetto voluto, perchè aumenta le dimensioni esterne dell'input. Credo che il bordo vada messo all'interno, in modo che le dimensioni del componente rimangano invariate.